package.json

```diff
@@ -59,7 +59,7 @@
   "dependencies": {
     "@actions/artifact": "^0.5.2",
     "@actions/core": "^1.4.0",
-    "@microsoft/powerplatform-cli-wrapper": "^0.1.36",
+    "@microsoft/powerplatform-cli-wrapper": "^0.1.38",
```

Comment:
in a second commit into this branch, update the dist/ folder (npm run update-dist)